manager/integration/tests/common.py

```diff
@@ -4555,3 +4561,42 @@ def restore_backup_and_get_data_checksum(client, core_api, backup, pod,
                                      'default')
 
     return data_checksum, output, restore_pod_name
+
+
+@pytest.fixture
+def nfs(request):
+
+    api = get_core_api_client()
+    apps_api = get_apps_api_client()
+
+    cmd = ["kubectl", "apply", "-f", LONGHORN_NFS_INSTALLATION_URL]
+    subprocess.check_output(cmd)
+
+    node_list = api.list_node()
+    for i in range(RETRY_COUNTS):
+        nfs_pods = {}
+        pod_list = api.list_namespaced_pod("default")
+
+        for pod in pod_list.items:
+            pod_name = pod.metadata.name
+            if "longhorn-nfs" in pod_name:
+                nfs_pods[pod_name] = \
+                    api.read_namespaced_pod(name=pod_name,
+                                            namespace='default').status.phase
```

Comment:
Probably you could consider using the label selector to filter longhorn-nfs Pods. For example
```shell
kubectl get pod -l app=longhorn-nfs-installation
```